sdk/python/kfp/cli/diagnose_me_cli.py

```diff
@@ -27,7 +28,7 @@ def diagnose_me():
 @click.option(
     '--namespace',
     type=Text,
-    help='Namespace to use for Kubernatees cluster. It will use environment default if not specified.'
+    help='Namespace to use for Kubernetes cluster. It will use environment default if not specified.'
 )
 @click.pass_context
 def diagnose_me(ctx, json, project_id, namespace):
```

Comment:
nit: `json` shadows outer package name `json`.

Reply:
good call! thank you.